fastcrud/crud/fast_crud.py

```diff
@@ -287,30 +290,61 @@ async def select(
         sort_orders: Optional[Union[str, list[str]]] = None,
         **kwargs,
     ) -> Selectable:
```

Comment:
I also missed some typing here : this should be of type `Select`, imported from `from sqlalchemy.sql.selectable import Select`